src/main/java/liquibase/ext/databricks/diff/output/changelog/ChangedTblPropertiesUtil.java

```diff
@@ -99,6 +102,14 @@ private static Map<String, String> convertToMap(Object referenceValueObject) {
                 .collect(Collectors.toMap(a -> a[0], a -> a[1]));
     }
 
+    /**
+     * Get the extended properties excluding delta parameters
+     */
+    public static String getExtendedProperties(String tblProperties) {
```

Comment:
Getting me bugged a bit, as far as we have an object `ExtendedTableProperties`, and this one represents filtered tblProperties. Could we rename this one?

Reply:
@SvampX, sorry, I will merge this ticket without this change to unblock testing of other tickets. Can we do this change is scope of some other PR?

Reply:
@SvampX you are right, naming it `getFilteredTblProperties` `removeDeltaPropertiesFromTblProperties` would make more sense?